Week-5/Exercise-5.4/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
this file should be in the root of the project.

Reply:
Actually, I used jest library for only a specific assignment so these files are in that assignments folder only.